src/grammar.coffee

```diff
@@ -35,28 +35,31 @@ class Grammar
   # @return [Array] An AST representing the unpacked expression.
   #
   @_unpack2DExpression: (expression) ->
-    mapping =
-      'bottom-left'   : ['left', 'bottom']
-      'bottom-right'  : ['right', 'bottom']
-      center          : ['center-x', 'center-y']
-      'intrinsic-size': ['intrinsic-width', 'intrinsic-height']
-      position        : ['x', 'y']
-      size            : ['width', 'height']
-      'top-left'      : ['left', 'top']
-      'top-right'     : ['right', 'top']
-
-    expressions = [expression]
-    property = expression[2]
-    properties = mapping[property]
+    # mapping =
+    #   'bottom-left'   : ['left', 'bottom']
+    #   'bottom-right'  : ['right', 'bottom']
+    #   center          : ['center-x', 'center-y']
+    #   'intrinsic-size': ['intrinsic-width', 'intrinsic-height']
+    #   position        : ['x', 'y']
+    #   size            : ['width', 'height']
+    #   'top-left'      : ['left', 'top']
+    #   'top-right'     : ['right', 'top']
+    #
+    # expressions = [expression]
+    # property = expression[2]
+    # properties = mapping[property]
+    #
+    # if properties?
+    #   expressions = []
+    #
+    #   for item in properties
+    #     expression = expression.slice()
+    #     expression[2] = item
+    #     expressions.push expression
 
-    if properties?
-      expressions = []
 
-      for item in properties
-        expression = expression.slice()
-        expression[2] = item
-        expressions.push expression
 
+    expressions = [expression]
     return expressions
```

Comment:
It looks like this method should be removed or renamed if this is all it's doing.